docs/reference/commandline/plugin_create.md

```diff
@@ -16,7 +16,7 @@ keywords: "plugin, create"
 # plugin create
 
 ```markdown
-Usage:  docker plugin create [OPTIONS] PLUGIN[:tag] PATH-TO-ROOTFS(rootfs + config.json)
+Usage:  docker plugin create [OPTIONS] PLUGIN PATH-TO-ROOTFS(rootfs + config.json)
```

Comment:
`PATH-TO-ROOTFS` probably isn't right, because it sounds like this should really be the parent directory of `rootfs`.